### ISSUE TITLE
Format the Profile Page

### ISSUE THREAD

**Issue by ZwoVierZwo** [friendica/friendica#12503]
The Profile Page is only Text.
Maybe it can be changed, so that we can use markdown for text format.

**Comment by MrPetovan** [comment]
Do you mean the profile description? You can already use BBCode tags.

**Comment by ZwoVierZwo** [comment]
Correct the description of me. 
Ok BBCode but markdown is not working - as i will work in the friendica posts.

**Comment by MrPetovan** [comment]
Indeed, however since profile settings aren't covered by a hook, the current Markdown addon cannot work its magic in this field yet.

**Comment by ZwoVierZwo** [comment]
so how about a parsing of the profile - cause i think the BBCode has already to be parsed and the converted the html description for display. 
I do not think this should be a big effort to do this?

**Comment by MrPetovan** [comment]
It isn't a big effort, but it is an effort. Here's what it entails:
1. Create a new hook
    1. Come up with a relevant name and a data structure
    1. Call it in the `Module/Settings/Profile/Index->post()` method
    1. Reference it in the documentation, both English and German (`doc/Addons.md` and `doc/de/Addons.md`)
1. Use it in the `markdown` addon
    1. Write the hook method using the data structure to convert the Markdown to HTML
    1. Register the hook in `markdown_install()` using the method name

**Comment by ZwoVierZwo** [comment]
Cool - thank you for the description - and Merry Christmas
